semgrep/semgrep/rule_match.py

```diff
@@ -97,11 +97,15 @@ def to_json(self) -> Dict[str, Any]:
         json_obj["extra"]["message"] = self._message
         json_obj["extra"]["metadata"] = self._metadata
         json_obj["extra"]["severity"] = self._severity
+
         if self._fix:
             json_obj["extra"]["fix"] = self._fix
+
         json_obj["start"] = self._start
         json_obj["end"] = self._end
-        json_obj["extra"]["lines"] = "\n".join(self.lines).rstrip()
+
+        if "line" in self.start and "line" in self.end:
+            json_obj["extra"]["lines"] = "\n".join(self.lines).rstrip()
```

Comment:
Should we put the regex match in "content" here? So it becomes one stop shop of the offending match?